src/Microsoft.AspNet.Mvc.ModelBinding/Binders/MutableObjectModelBinder.cs

```diff
@@ -310,21 +316,12 @@ private static bool TryGetPropertyDefaultValue(PropertyInfo propertyInfo, out ob
             }
         }
 
-        internal static PropertyValidationInfo GetPropertyValidationInfo(ModelBindingContext bindingContext)
+        private static PropertyValidationInfo GetPropertyBindingInfo(ModelBindingContext bindingContext)
```

Comment:
In some past version this was internal for testing (possibly in webapi), but there are no tests that use it in the current codebase.